app/v2/generators/DepartureMessageGenerator.scala

```diff
@@ -545,4 +550,189 @@ class DepartureMessageGeneratorImpl @Inject()(clock: Clock) extends Generators w
         </HolderOfTheTransitProcedure>
       </ncts:CC140C>
     )
+
+  private def generateIE019Message(correlationId: String): XMLMessage =
+    XMLMessage(
+      <ncts:CC019C PhaseID="NCTS5.0" xmlns:ncts="http://ncts.dgtaxud.ec">
+        <messageSender>{Strings.alphanumeric(1, 35)}</messageSender>
```

Comment:
Strings is already imported on top. Please remove and use {alphanumeric(1, 35)}. Please do it for other places as well